terraform/dev/account/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "subject_claim_filters" {
```

Comment:
Do all of these need to be variables, or could you just put them inline in `github-oidc.tf`? I think these values won't change often and I don't think this module will be consumed elsewhere. If you want to keep them as variables I think that getting rid of the defaults and setting the values in a `terraform.tfvars` file is clearer

Reply:
I agree. I like being clear. I removed the variables. All the variables. I put the values in-line.
BTW, AWS says thumbprints are no longer used with github provider (but the attribute is still required by terraform).
I removed all variables. `variables.tf` is no longer necessary.